Firestore/core/test/firebase/firestore/local/local_serializer_test.cc

```diff
@@ -216,14 +221,32 @@ TEST_F(LocalSerializerTest, EncodesNoDocumentAsMaybeDocument) {
   ExpectRoundTrip(no_doc, maybe_doc_proto, no_doc.type());
 }
 
+// TODO(rsgowman): Requires held write acks, which aren't fully ported yet. But
+// it should look something like this.
+#if 0
+TEST_F(LocalSerializerTest, EncodesUnknownDocumentAsMaybeDocument) {
```

Comment:
might be able to use DISABLE_EncodesUnknownDocumentAsMaybeDocument to annotate the disable?

Reply:
It's more than just disabling it; the code won't even compile since the objects under test don't actually exist yet. Technically, I should just delete this entirely, since it has nothing to do with this PR, but since I accidentally wrote it before realizing that it requires held write acks, I figured I'd just leave it around as a placeholder.